config/client_config/me/system/config/templates/features/enroll_app/fdsh_services.yml

```diff
@@ -19,3 +19,8 @@ registry:
         is_enabled: <%= ENV['RENEWAL_ELIGIBILITY_VERIFICATION_USING_RRV_IS_ENABLED'] || true %>
       - key: :validate_and_record_publish_errors
         is_enabled: <%= ENV['VALIDATE_AND_RECORD_PUBLISH_ERRORS_IS_ENABLED'] || false %>
+      - key: :validate_and_record_publish_application_errors
+        is_enabled: <%= ENV['VALIDATE_AND_RECORD_PUBLISH_APPLICATION_ERRORS_IS_ENABLED'] || false %>
+        settings:
```

Comment:
We will not be using csr_codes and instead, the logic related to enrollment will be reused.